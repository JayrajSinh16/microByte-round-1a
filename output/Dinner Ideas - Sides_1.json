{
  "title": " Ingredients :",
  "outline": [
    {
      "level": "H1",
      "text": "Agedashi Tofu",
      "page": 2
    },
    {
      "level": "H2",
      "text": " Instructions :",
      "page": 4
    },
    {
      "level": "H3",
      "text": " Instructions :",
      "page": 14
    }
  ]
}