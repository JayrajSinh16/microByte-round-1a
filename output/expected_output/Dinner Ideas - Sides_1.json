{
    "title": "Dinner - Sides",
    "outline": [
        {
            "level": "H1",
            "text": "Acaraj√©",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Agedashi Tofu",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Aloo Gobi",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Albondigas",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 3
        },
        {
            "level": "H1",
            "text": "Antipasto",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 3
        },
        {
            "level": "H1",
            "text": "Arancini",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Baba Ganoush",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Baked Beans",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Bammy",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Banchan",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Banh Beo",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Banh Xeo",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Basturma",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Batbout",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Bhindi Masala",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 7
        },
        {
            "level": "H1",
            "text": "Bobotie",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 7
        },
        {
            "level": "H1",
            "text": "Bok Choy with Garlic",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Bolinho de Bacalhau",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Bouillabaisse",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Bratkartoffeln (Fried Potatoes)",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 9
        },
        {
            "level": "H1",
            "text": "Briam",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 9
        },
        {
            "level": "H1",
            "text": "Briouats",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 10
        },
        {
            "level": "H1",
            "text": "Bruschetta",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 10
        },
        {
            "level": "H1",
            "text": "Bulgur Pilaf",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 10
        },
        {
            "level": "H1",
            "text": "Ca Tim Nuong (Grilled Eggplant)",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Callaloo",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Canh Chua",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 12
        },
        {
            "level": "H1",
            "text": "Caprese Salad",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 12
        },
        {
            "level": "H1",
            "text": "Carrot Salad",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 12
        },
        {
            "level": "H1",
            "text": "Ceviche",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Cha Gio",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Chakchouka",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Chana Masala",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Chapati",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Char Siu",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        },
        {
            "level": "H1",
            "text": "Chawanmushi",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        },
        {
            "level": "H1",
            "text": "Chiles Rellenos",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        }
    ]
}